src/Microsoft.DotNet.Interactive.CSharpProject/RoslynWorkspaceUtilities.cs

```diff
@@ -0,0 +1,389 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Text;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using static Microsoft.DotNet.Interactive.CSharpProject.BuildCacheFileUtilities;
+
+namespace Microsoft.DotNet.Interactive.CSharpProject;
+
+public static class RoslynWorkspaceUtilities
+{
+    public class ProjectItem
+    {
+        public string ItemSpec { get; }
+        public IReadOnlyDictionary<string, string> Metadata { get; }
+
+        internal ProjectItem()
+        {
+//            ItemSpec = taskItem.ItemSpec;
+//            Metadata = taskItem.MetadataNames.Cast<string>().ToDictionary(x => x, x => taskItem.GetMetadata(x));
+        }
+    }
+
+    public class BuildDataResults
+    {
+        public BuildProjectData Data { get; set; }
```

Comment:
I think it's a little clearer if the property name matches the type name.

Reply:
Done.

Renamed 'Data' with 'BuildProjectData'